### ISSUE TITLE
Visual Studio Community 2022 / 2019 doesn't install

### ISSUE THREAD

**Issue by scowalt** [anderlli0053/DEV-tools#66]
## Bug Report

**Package Name:** `MicrosoftVisualStudioCommunity2019-Install`

### Current Behaviour

Puts the Visual Studio installer on your machine, but doesn't install Visual Studio 2019.

### Expected Behaviour

Should download and install Visual Studio 2019. (Probably to the default install location with no workflows added)

### System details

**Windows version:** Windows 11 (22621.963)

**OS architecture:** 64bit

**PowerShell version:** 5.1.22621.963

#### Scoop Configuration

```json
{
    "last_update":  "2022-12-20T11:07:58.7728710-08:00",
    "scoop_branch":  "master",
    "scoop_repo":  "https://github.com/lukesampson/scoop"
}
```

**Comment by anderlli0053** [comment]
ACooper who wrote the original manifests said this about 5 hours ago, when i mentioned this issue with the manifests...:

`I have made an update to both MicrosoftVisualStudioCommunity2019-Install and MicrosoftVisualStudioCommunity2022-Install manifests. I have tested both installs and they are installing Visual Studio Installer and Visual Studio 2019/2022 Community. They still come up with errors but they seem to be working now. You will still need to run Visual Studio Installer after the install finishes to add any required workloads.`

So i'm guessing that's it and when you try to install it it still comes up with **Exit Code 81** and **Exit Code -1**, and i guess i'm closing this now as of this time there is no complete solution to install Visual Studio(s) via Scoop bucket manifests.. I may reopen the issue sometime in the future if this will be possible and without error and such.
